Utilities/Thread.cpp

```diff
@@ -1761,7 +1761,7 @@ static LONG exception_filter(PEXCEPTION_POINTERS pExp) noexcept
 
 const bool s_exception_handler_set = []() -> bool
 {
-	if (!AddVectoredExceptionHandler(1, (PVECTORED_EXCEPTION_HANDLER)exception_handler))
+	if (!AddVectoredExceptionHandler(FALSE, (PVECTORED_EXCEPTION_HANDLER)exception_handler))
```

Comment:
Wouldn't making our exception handler last degarde performance?

Reply:
Yea, this seems like something that should be enabled behind a compile time flag. We don't need to ship asan support on release binaries.

Reply:
Why would it matter with the only handler in said release?

Reply:
> Why would it matter with the only handler in said release?

Without this change ASAN is not working on Windows. I found the solution here https://developercommunity.visualstudio.com/t/asan-problems-when-manual-exception-handling/1242524#T-N1295191 .

Is `USE_ASAN` OK as compile time flag?

Reply:
Added `USE_ASAN` flag.